src/nimble.nim

```diff
@@ -673,6 +673,28 @@ proc execBackend(options: Options) =
         [backend, args, bin], showOutput = true)
   display("Success:", "Execution finished", Success, HighPriority)
 
+proc tempOutArg(file: string): string =
```

Comment:
Is this so that binaries are not created inside the user's ``tests`` directory? I wonder if it would be useful to not do that.